loony_dev/web/static/app.css

```
@@ -1030,6 +1030,54 @@ button.action:hover,
+  font: inherit;
+  background: var(--surface);
+  border: 1px solid color-mix(in srgb, var(--state-blocked) 32%, var(--border));
+  border-radius: var(--radius);
+  box-shadow: var(--shadow);
+  padding: var(--sp-3);
+  cursor: pointer;
+}
+.triage-card:hover { border-color: var(--accent); }
+.triage-card:focus-visible { outline: 2px solid var(--accent); outline-offset: 2px; }
+.triage-title { font-weight: 600; line-height: 1.3; }
+.triage-title-text { word-break: break-word; }
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)